app/pages/patient/patientinfo.js

```diff
@@ -252,45 +228,51 @@ var PatientInfo = translate()(React.createClass({
         {this.renderDataSources()}
       </div>
     );
-  },
+  }
 
-  renderFullNameInput: function(formValues) {
-    var fullNameNode, errorElem, classes;
-    var error = this.state.validationErrors.fullName;
+  renderFullNameInput(formValues) {
+    let fullNameNode, errorElem, classes;
+    const errors = {
+      firstName: this.state.validationErrors.firstName,
+      lastName: this.state.validationErrors.lastName
+    };
     // Legacy: revisit when proper "child accounts" are implemented
     if (personUtils.patientIsOtherPerson(this.props.patient)) {
       classes = 'PatientInfo-input';
-      if (error) {
+      if (errors.firstName || errors.lastName) {
         classes += ' PatientInfo-input-error';
-        errorElem = <div className="PatientInfo-error-message">{error}</div>;
+        errorElem = <div className="PatientInfo-error-message">
+            {errors.firstName?errors.firstName:''}
+            <br/>
+            {errors.lastName?errors.lastName:''}
+          </div>;
       }
       fullNameNode = (
-        <div className={classes}>
-          <input className="PatientInfo-input" id="fullName" ref="fullName" placeholder="Full name" defaultValue={formValues.fullName} />
+        <div className="PatientInfo-blockRow">
+          <input className={classes} id="firstName" ref="firstName" placeholder="First name" defaultValue={formValues.firstName} />
+          <input className={classes} id="lastName" ref="lastName" placeholder="Last name" defaultValue={formValues.lastName} />
           {errorElem}
         </div>
       );
     }
     else {
-      formValues = _.omit(formValues, 'fullName');
+      formValues = _.omit(formValues, ['firstName','lastName']);
       const fullName = this.getDisplayName(this.props.patient);
       fullNameNode = (
-        <Trans className="PatientInfo-block PatientInfo-block--withArrow" i18nKey="html.patient-info-fullname">
-          {{fullName}} (edit in
-          <Link to="/profile">account</Link>)
-        </Trans>
+        <div className="PatientInfo-block PatientInfo-block--withArrow">
+          {fullName} ({t('edit in')} <Link to="/profile">{t('account')}</Link>)
+        </div>
       );
     }
 
     return (<div className="PatientInfo-blockRow">
       {fullNameNode}
     </div>);
-  },
+  }
 
-  renderBirthdayInput: function(formValues) {
-    const { t } = this.props;
-    var classes = 'PatientInfo-input', errorElem;
-    var error = this.state.validationErrors.birthday;
+  renderBirthdayInput(formValues) {
+    let classes = 'PatientInfo-input', errorElem;
+    const error = this.state.validationErrors.birthday;
```

Comment:
For some reason, I wasn't able to modify this value

Reply:
Date format with En user ?

Reply:
For now, YLP will be released in European countries that seem to use DMY so let's keep this format (cF https://en.wikipedia.org/wiki/Date_format_by_country).

Reply:
For now, I don't think we make any distinction between language/country. 
So en language is en-us (inherited from Tidepool) and date format is for now MM/DD/YYYY. 

We should introduce en-gb language to manage this properly...